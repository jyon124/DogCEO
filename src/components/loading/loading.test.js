import React from 'react';
import ReactDOM from 'react-dom';
import Loading from './loading.js';

import Enzyme, { shallow, render, mount } from 'enzyme';
import toJson from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';

Enzyme.configure({ adapter: new Adapter() })
describe('Loading', () => {
    it('renders a Loading component', () => {
        const wrapper = shallow(<Loading />)
        expect(toJson(wrapper)).toMatchSnapshot();
    });
});