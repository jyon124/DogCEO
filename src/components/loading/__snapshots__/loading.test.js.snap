// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading renders a Loading component 1`] = `
<Fragment>
  <div
    className="lds"
  >
    <div />
    <div />
    <div />
  </div>
</Fragment>
`;
